### DISCUSSION TITLE
Event counter can overflow and crash server

### DISCUSSION THREAD

**Discussion by Roanoke23** [FakeFishGames/Barotrauma#15335] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

"event counter can overflow and crash server" I was playing a game with my friends, and something is making it crash. I have no idea what mod it is, it might not even be a mod at this point. It just makes the server crash over and over again. Apperently something is spamming an input or some kind of "event" that makes it crash. 

### Reproduction steps

![image](https://github.com/user-attachments/assets/9053d1df-240f-49a2-aa59-9546e96fd166)
![image](https://github.com/user-attachments/assets/af30866e-eb13-4556-9324-cc970a139315)
![image](https://github.com/user-attachments/assets/44a19725-844a-4432-ba19-8577e299ea45)
![image](https://github.com/user-attachments/assets/2019e959-170e-4173-83e5-c8c9bf056046)
These are the mods I was using, I removed more, but this did not help with the server not crashing.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

To be clear, does the server actually crash (do you see a message box saying the server crashed), or does the client just get kicked to the main menu with some kind of an error message? Would it be possible to send us the crash report (if there is one) or a screenshot of the error message(s) you're getting? Thanks and sorry for the inconvenience!
